src/app/components/Button/KeyBox/KeyBox.js

```diff
@@ -43,16 +43,16 @@ function KeyBox(props) {
   return (
     <Box ref={hoverRef}>
       <Button buttonType="secondary" onClick={() => onButtonClick(letter)}>
-        {item?.choice?.image && (
-          <Row>
-            <Col>
-              <img
-                alt=""
-                className="imageContainer"
-                src={item?.choice?.image || './public/default.png'}></img>
-            </Col>
-          </Row>
-        )}
+        {/*{item?.choice?.image && (*/}
```

Comment:
Remove this comments. I'll add right code later.

Reply:
fixed